python/plugins/processing/gui/ExtentSelectionPanel.py

```diff
@@ -205,10 +152,9 @@ def setValueFromRect(self, r):
         self.dialog.activateWindow()
 
     def getValue(self):
-        if unicode(self.leText.text()).strip() != '':
-            return unicode(self.leText.text())
-        else:
-            return self.getMinCoveringExtent()
+        if unicode(self.leText.text()).strip() == '':
+            return None
```

Comment:
If this is None, how is that then set in the parameter. By looking at https://github.com/qgis/QGIS/blob/master/python/plugins/processing/core/parameters.py#L197, it wil return False, unless the parameter is optional (which is not by default). Can you make the changes in the parameters module to fix that? thanks!